### PR TITLE
Updates to support importing confidence intervals (issue #151)

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: viewer/api/models.py [hunk 1]
```diff
@@ -167,4 +167,7 @@ class Stats(models.Model):
     population = models.CharField(max_length=255, default='')
     subgroup = models.CharField(max_length=255, default='')
     key = models.CharField(max_length=100)
-    value = models.IntegerField()
+    value = models.FloatField(blank=True, null=True)
```
Comment (sunahsuh) [inline viewer/api/models.py]: Do we really want blank/null in `value`? I think if we don't have one, something has gone pretty wonky.

File: notebooks/import.py [hunk 2]
```diff
@@ -240,11 +240,13 @@ def create_points(cursor, collection_id, histogram):
 
 
 def create_stat(cursor, dataset_id, metric_id, population, subgroup, key,
-                value):
+                value, low=None, high=None, level=None):
```
Comment (sunahsuh) [inline notebooks/import.py]: Probably makes sense to specify `confidence_*` -- it's not clear from the context of just the method signature what low/high/level might mean

File: viewer/api/models.py (hunk 1 above)
Comment (robhudson) [reply viewer/api/models.py]: This field is also overloaded for the general branch stats, like total pings and total clients so I was being lenient here. But now that you mention it those should also always have values.

File: notebooks/import.py (hunk 2 above)
Comment (robhudson) [reply notebooks/import.py]: :+1:

**Pull request merged**